submissions/end-2021/end-2021.md

```diff
@@ -0,0 +1,84 @@
+
+
+One of my favourite mechanics pre 0.15 was that you could void any fluid in steam engines. So when setting up oil. you didn't need to crack the other liquids - you simply voided them in steam engines and had a very easy setup. There was also the possibility to place offshore pumps for water at the edge of the map. So we speedrunners did limit the map on purpose to generate a straight line for a powerplant at a desired position. Also not all seeds were different at the old times, instead you had one big master map and the seed just determinised your starting position on it. So we speedrunners kind of generated the master map and chose the seed accordingly, so you start at a desired location.
+
+When I ran 100% back in the days it required farming thousands of [alien goop](https://alt-f4.blog/ALTF4-6/#alien-artifacts-a-primitive-technology). It was really annoying. I literally went alien hunting for over 2 hours that days just annihilating the biters without any skill requiered. Really good they removed this
+
+![AntiElitz first Any% speedrun upload to YouTube in 2015. It was the world record at 3:45:58](https://youtu.be/cCiG-7gLTcg)
+
+## The Highlights of Factorio history <author>[MojoD](https://www.twitch.tv/mojod)</author>
+
+Since 0.12 two releases major releases really stand out above all others, 0.13 and 0.15.  These two releases had a lot to look forward to and they delivered the foundation of what we know of the game today.
+
+0.13 was the a big step up in quality of life and addressed many of the frustrating issues with trains and combat (who remembers having to constantly cycle the 8 rotations of curved rail), removal of smart inserter and smart chest and removing a lot of annoying limitations of the circuit network as well and most significant of all buildings taking damage no longer had an alert sound.  There was also hope of seeing the Spidertron in game we were teased with months prior.  As addictive as the game was before 0.13 the game was pretty unbearable at times.
+
+Of all releases 0.15 I feel was the most anticipated and significant update as it completely transformed the game and was the foundation of what became the game is now.  Among the other changes the upgraded tech tree finally giving purpose to end game with infinite science.  Unfortunately releases at this stage never came out on time and would finally be released after months of delays.
+
+The release also stands out because by the power of some creative modding, a Colonelwill and the moral support from a copper patch that bore remarkable resemblance to a duck a it was possible to build 1K Spm factory producing 0.15 infinite science in 0.14.  It also shows how far above and beyond Wube has gone over the years because not only did the 0.14 factory work in 0.15, the save and the mod also can be brought forward to the latest 1.1 release and still work as it should. 
```

Comment:
```suggestion
The release also stands out, because with the power of some creative modding, a ColonelWill, and the moral support from a copper patch that bore remarkable resemblance to a duck, it was possible to build a 1K SPM factory producing 0.15's space science packs in 0.14. It also shows how far above and beyond Wube has gone over the years, because not only did the 0.14 factory work in 0.15, but the save and the mod can also be updated to the latest 1.1 release and they'll still work as they should. 
```